src/components/AddressForm.js

```diff
@@ -0,0 +1,192 @@
+import {CONST as COMMON_CONST} from 'expensify-common/lib/CONST';
+import lodashGet from 'lodash/get';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import React, {useCallback} from 'react';
+import _ from 'underscore';
+import AddressSearch from './AddressSearch';
+import CountrySelector from './CountrySelector';
+import Form from './Form';
+import StatePicker from './StatePicker';
+import TextInput from './TextInput';
+import useLocalize from '../hooks/useLocalize';
+import CONST from '../CONST';
+import * as ValidationUtils from '../libs/ValidationUtils';
+import styles from '../styles/styles';
+import Navigation from '../libs/Navigation/Navigation';
+
+const propTypes = {
+    city: PropTypes.string,
+    country: PropTypes.string,
+    formID: PropTypes.string.isRequired,
+    onAddressChanged: PropTypes.func,
+    onSubmit: PropTypes.func.isRequired,
+    state: PropTypes.string,
+    street1: PropTypes.string,
+    street2: PropTypes.string,
+    submitButtonText: PropTypes.string,
+    zip: PropTypes.string,
```

Comment:
These please all need some comments above them like we have in other parts of the app.

Reply:
@pac-guerreiro could you address this comment